scripts/common/setup_env.sh

```diff
@@ -45,24 +45,27 @@ else
 fi
 
 # Update IPDK RECIPE libraries
-export LD_LIBRARY_PATH=$IPDK_RECIPE/install/lib:$IPDK_RECIPE/install/lib64:$LD_LIBRARY_PATH
-export PATH=$IPDK_RECIPE/install/bin:$IPDK_RECIPE/install/sbin:$PATH
+export LD_LIBRARY_PATH=$P4CP_INSTALL/lib:$P4CP_INSTALL/lib64:$LD_LIBRARY_PATH
+export PATH=$P4CP_INSTALL/bin:$P4CP_INSTALL/sbin:$PATH
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib64
 
 # Update Dependent libraries
-export LD_LIBRARY_PATH=$DEPEND_INSTALL/lib:$DEPEND_INSTALL/lib64:$LD_LIBRARY_PATH
-export PATH=$DEPEND_INSTALL/bin:$DEPEND_INSTALL/sbin:$IPDK_RECIPE/install/sbin:$IPDK_RECIPE/install/bin:$PATH
-export LIBRARY_PATH=$DEPEND_INSTALL/lib:$DEPEND_INSTALL/lib64:$LIBRARY_PATH
+export LD_LIBRARY_PATH=$P4CP_DEP_INSTALL/lib:$P4CP_DEP_INSTALL/lib64:$LD_LIBRARY_PATH
+export PATH=$P4CP_DEP_INSTALL/bin:$P4CP_DEP_INSTALL/sbin:$P4CP_INSTALL/sbin:$P4CP_INSTALL/bin:$PATH
+export LIBRARY_PATH=$P4CP_DEP_INSTALL/lib:$P4CP_DEP_INSTALL/lib64:$LIBRARY_PATH
 
 echo ""
 echo ""
 echo "Updated Environment Variables ..."
 echo "SDE_INSTALL: $SDE_INSTALL"
```

Comment:
I'd put `SDE_INSTALL` and `DEPEND_INSTALL` together.

Is `P4CP_INSTALL` significant enough to be worth displaying?

Reply:
yes that's where all P4CP libraries and binaries are avaiable. For example, infrap4d. p4rt-ctl etc are available under P4CP_INSTALL

Reply:
Oh. Duh.